app/assets/javascripts/application.js

```diff
@@ -158,6 +158,68 @@ $(document).on('hidden.bs.modal', function(e) {
     OPEN_MODAL = '';
 });
 
+/**
+ * Logs properties about rendered Study Overview plot to Mixpanel via Bard.
+ *
+ * See overview diagram at:
+ * https://app.lucidchart.com/invitations/accept/3b086049-89bc-4db2-96c3-815ccc438bd5
+ *
+ * @param {String} plotType Type of plot, e.g. scatter, violin, box, dot
+ */
+function logPlot(plotType) {
+
+  var endTime = performance.now();
+  var startTimeFrontend = window.SCP.perfTimeStartPlotRender;
```

Comment:
how about creating a sub-scope within window.SCP for perfTime stuff?  So this could be window.SCP.perf.startPlotRender?

Reply:
Per our chat, it sounds like your proposed refinement would address this, and I'll review when it's ready.